airbyte-integrations/connectors/source-amazon-seller-partner/metadata.yaml

```diff
@@ -20,6 +20,11 @@ data:
     oss:
       enabled: true
   releaseStage: alpha
+  releases:
+    breakingChanges:
+      2.0.0:
+        message: Brand Analytics Reports will be removed from Cloud verison. Deprecated FBA reports will be removed permanently.
+        upgradeDeadline: "2023-11-24"
```

Comment:
let's make the date 11-29 we have an american holiday next week

Reply:
done